### PR TITLE
Enable tests for alterations Read and update - backward compatibility

### PR DESCRIPTION
**Pull request by sahithikkss-zz**
Enable alterations test for V1 resource using preview sdk
Ensures backward compatibility

### CODE REVIEW

File: sdk/cognitiveservices/Knowledge.QnAMaker/tests/BaseTests.cs [hunk 1]
```diff
@@ -12,15 +12,15 @@ public abstract class BaseTests
         static BaseTests()
         {
             // Retrieve the configuration information.
-            QnAMakerSubscriptionKey = "";
+            QnAMakerSubscriptionKey = "7afcaa16ba1747328eb7a77de8316445";
```
Comment (somigithub) [inline sdk/cognitiveservices/Knowledge.QnAMaker/tests/BaseTests.cs:15]: you want to reset this?

**Review by somigithub — CHANGES_REQUESTED** [review]

File: sdk/cognitiveservices/Knowledge.QnAMaker/tests/BaseTests.cs (hunk 1 above)
Comment (sahithikkss) [reply sdk/cognitiveservices/Knowledge.QnAMaker/tests/BaseTests.cs]: sure. thanks

**Comment by sahithikkss** [comment]
@AlexGhiondea could you please review, approve and merge these changes as applicable?

**Review by somigithub — APPROVED** [review]

**Review by AlexGhiondea — APPROVED** [review]

**Comment by sahithikkss** [comment]
@AlexGhiondea Could you please merge this as well?

**Pull request merged**